src/Sylius/Behat/Element/Product/ShowPage/MediaElement.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Product\ShowPage;
+
+use Behat\Mink\Element\NodeElement;
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class MediaElement extends Element implements MediaElementInterface
+{
+    public function isImageDisplayed(): bool
+    {
+        $imageElement = $this->getDocument()->find('css','#media a img');
```

Comment:
Maybe we could move `sprintf('#pricing tr:contains("%s")', $channelName)` to getDefinedElements() function, like:

```
'image' => '#media a img',
```

Reply:
I do not agree with you, in this case, `getDocument` take just one line of code, `getElement` make code less readable - "less is more"